docs/blueprints/EP031.rst

```diff
@@ -0,0 +1,273 @@
+
+To enable INT, first a physical loop has to be deployed. For this example, on INT Hop Z, port 1 is connected to port 2 by a physical fiber patch cord (done during commissioning). Then, the following flows need to be ADDED to the pipeline:
+
+  1. Adding INT metadata:
+
+    - match:
+
+      - table 0
+
+        - priority: Higher (40000)
+        - match: <in_port= **11**,vlan_vid=20,eth_type=0x800,ip_proto=6>  # TCP
+        - instrutions:
+
+          - action: add_int_metadata
+          - action [set_queue, pop_vlan, etc.]
+          - action: output port **1** (loop)
+
+        - priority: Higher (40000)
+        - match: <in_port= **11**,vlan_vid=20,eth_type=0x800,ip_proto=17>. # UDP
+        - instrutions:
+
+          - action: add_int_metadata
+          - action [set_queue, pop_vlan, etc.]
+          - action: output port **1** (loop)
+
+  2. Send Report and pop INT data (traffic is coming from port 2 that's the loop with port 1). Only INT data gets into the loop.
+
+    - match:
+
+      - table 0
+
+        - priority: Higher (40000)
+        - match: <in_port= **2**,vlan_vid=20>
+        - instrutions:
+
+          - action: send_report
+          - action go to table 2
+
+
+      - table 2
+
+        - priority: Higher (40000)
+        - match: <in_port= **2**,vlan_vid=20>
+        - instrutions:
+
+          - action: pop_int
+          - action [set_queue, pop_vlan, etc.]
+          - action: output port **23** (original port)
+
+
+IV. How to enable INT for EVCs
+==============================
+
+The goal for the **telemetry** app is to enable telemetry for ALL EVCs. However, it must support enabling and disabling telemetry for a single EVC or ALL EVCs. This is the approach:
+
+  1 . The **telemetry** napp will start operating once **mef_eline** is loaded and EVCs and their flows are pushed to the data plane.
+  2. **telemetry** will listen for events *kytos/mef_eline.(redeployed_link_(up|down)|deployed)* and *kytos.mef_eline.created* issued by **mef_eline**.
+  3. For each EVC identified, **telemetry** will
+    1. use EVC's cookie to get all flow entries created by **flow_manager** IF telemetry is not already enabled.
+    2. push more specific flows as described in Section II
+    3. add a key in the EVC's metadata called "telemetry" with value "enabled". key "telemetry" will be "disabled" once telemetry is disabled for an EVC.
+
+V. Events
+==========
+
+  1. Listening
+    1. *kytos/mef_eline.(redeployed_link_(up|down)|deployed)*
+    2. *kytos.mef_eline.created*
+
+  2. Issuing
+    1.  *kytos.telemetry.enabled*
+    2.  *kytos.telemetry.disabled*
+
+
+VI. REST API
+=============
+
+  - /telemetry/v1/evc/enable - POST - REST to enable/create INT flows for an EVC_ID.
+  - /telemetry/v1/evc/disable - POST - REST to disable/remove INT flows for an EVC_ID.
+  - /telemetry/v1 - GET - List all INT-enabled EVCs
+  - /telemetry/v1/proxies - GET - REST to return the list of proxy ports in the topology
```

Comment:
@jab1982 if, in this iteration, strategically, there's no concern for proxies then it's worth segregating future potential endpoins and what's going to be implemented with a basic spec and how the response, body and args should look like, that's very helpful for development and code review too.